static/js/manage/manage-searches.js

```diff
@@ -0,0 +1,48 @@
+MEDUSA.manage.manageSearches = function() {
+    // Get total number current scene exceptions per source. Will request medusa, xem and anidb name exceptions.
+    var url = 'manage/searches/sceneexception/retrieve';
+    var updateExceptionTable = function(exceptions) {
+        var status = $('#sceneExceptionStatus');
+
+        var table = $('<ul></ul>')
```

Comment:
I tried to improve it. But some lines need to be broken, to prevent them from getting too long.